### DISCUSSION TITLE
Unsupported ODS issue: found 11.2 support 13.1

### DISCUSSION THREAD

**Discussion by tester0077** [mariuz/flamerobin#371] [General]
Quite some time ago I had one of my utilities working with a Firebird embedded DB. 
After a OS crash/re-installation,  I had to work on the app again, but I can no longer open the DB from my earlier version. 
FlameRobin (FR) complains about an "unsupported ODS structure".
I am at a total loss as to how to recover and proceed from here and would very much appreciate some guidance.
Another recent discussion talks about the cause being a 'wrong' version of fbclient'.
A quick check shows I have at least 6 or seven files with the name fbclient.dll in various places on my main drive.
1) which version do I need?
2) how do I determine the version of a fbclient.dll?
3) where does it have to reside to be of use to FR?
4) how can I make sure both FR & my app use the same fbclient.dll - and any related/necessary files (firebird.msg??)
Any help will be very much appreciated.
TIA

**Comment by arvanus** [comment]
https://ib-aid.com/en/articles/all-firebird-and-interbase-on-disk-structure-ods-versions/
So, looks like your database where created by Fb2.5, and your server are 5.0
You need to run an Fb2.5 server OR use and fbembed.dll from 2.5 to access this DB

**Comment by tester0077** [comment]
Thank you for the feedback, but my real problems seems to be:
1) how to test for and/or make sure the DLL version selected by/for the user is the correct one.
2) how to even, hopefully, allow the user to update the database to avoid the problem every time he needs to read the data.

**Comment by mariuz** [comment]
@tester0077 please use Firebird support for these kind of questions (describe there the issues that you have)
Thank you

https://groups.google.com/g/firebird-support
